internal/templating/main.go

```diff
@@ -7,9 +7,41 @@ import (
 	"text/template"
 	"time"
 
+	"github.com/natesales/bcg/internal/config"
 	"github.com/joomcode/errorx"
 )
 
+type Neighbor struct {
+	Address string
+	Name string
+	Protocols []string
+}
+
+func ConvertNeighborsToStruct(Neighbors []string, mp bool) []Neighbor {
+	neighbors := []Neighbor{}
+	for _, neighbor := range Neighbors {
+		Name := "MP"
+		if !mp {
+			if strings.Contains(neighbor, ":"){
+				Name = "6"
+			} else {
+				Name = "4"
+			}
+		}
+		Protos := []string{Name}
+		if Name == "MP" {
+			Protos = []string{"4", "6"}
```

Comment:
What is this for? Should this be a `multiprotocol` boolean instead and default that to IPv4/IPv6 AFIs?